packages/tempus/test/Subcycling/Tempus_SubcyclingTest.cpp

```diff
@@ -241,6 +245,9 @@ TEUCHOS_UNIT_TEST(Subcycling, SinCosAdapt)
     stepper->setSubcyclingMaxFailures      (10);
     stepper->setSubcyclingMaxConsecFailures(5);
     stepper->setSubcyclingScreenOutputIndexInterval(1);
+    //stepper->setSubcyclingIntegratorObserver(
+    //  Teuchos::rcp(new Tempus::IntegratorObserverSubcycling<double>()));
+    //stepper->setSubcyclingPrintDtChanges   (true);
```

Comment:
Are these comments needed anymore?

Reply:
I kept these for when we want to output the subcycling timestep information during debugging.